### PR TITLE
Fixes getting the default user language from the OS

### PR DESCRIPTION
**Pull request by kdschlosser**
default language revisited

This is the second time doing this. the first time I could not remember
what the problem was exactly. It is something that is not easily
replicated.

Details:

the description of 

    wx.Locale.GetSystemLanguage

states "tries to detect users default locale setting."
This description is inaccurate. It does not try to get the "users"
default locale. it does exactly what the method name implies.
it gets the "system" default language. which may not be the language
the currently logged in user has set to be displayed o their screen.

The system default language is determined by what was purchased. So if
i live in Russia. the Windows available for purchase there would be a
Russian edition. Now as the user I can tell windows to display whatever
language I want. so to me everything is pretty seamless. If we
installed EG on a Russian copy of Windows where the user has all of the
language settings set to English, EG would display the Russian language.
Not Ideal. We want to display the language that the user is
currently displaying in Windows.

updates option dialog to display language selection in native tongue.

### CODE REVIEW

**Comment by kdschlosser** [comment]
The last PR fixes #319

File: eg/Classes/Translation.py [hunk 1]
```diff
@@ -16,145 +16,1015 @@
...
+        'br_BR': u'Breton',
+        'bs_BS': u'Bosanski',
+        'ca_CA': u'Català',
+        'co_CO': u'Corsican',
+        'cs_CS': u'čeština',
+        'cy_CY': u'Cymraeg',
+        'da_DA': u'dansk',
+        'de_AT': u'Deutsch - Österreich',
+        'de_CH': u'Deutsch - Schweiz',
+        'de_DE': u'Deutsches Deutschland',
```
Comment (topic2k) [inline eg/Classes/Translation.py]: It's just
```suggestion
        'de_DE': u'Deutsch',
```

**Comment by kdschlosser** [comment]
I did think of this. But if you follow the rest of the German layouts like 
German - Austrian
German - Switzerland

the German - Germany would be correct no? I am not sure if the syntax is correct. i do remember seeing in the translator that it was a confirmed syntax. But you would know better then I, seeing how you live there and all ¯\_(ツ)_/¯

**Comment by topic2k** [comment]
Ok, if you want to follow the syntax then "Deutsch - Deutschland" would be the correct spelling. I'm ok with either variant.

**Comment by topic2k** [comment]
What's your opinion? Will you change `Deutsches Deutschland` to one of 
- `Deutschland`
- `Deutsch - Deutschland`
- `Deutsch`

**Comment by kdschlosser** [comment]
Which ever way you want. I am not able to make the change. Or I should say I don't want to. I am on my cellphone it will be a pain.  Can you make the change?

**Comment by topic2k** [comment]
Ok, will do.

**Pull request merged**